examples/parquet_analytics_demo.ipynb

```diff
@@ -0,0 +1,476 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "af5806d3",
+   "metadata": {},
+   "source": [
+    "# Parquet Analytics Demo Notebook"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6f5d5a85",
+   "metadata": {},
+   "source": [
+    "This notebook serves as a brief demonstration of how a set of cloud-hosted parquet files can be analyzed locally using R. While the analysis in this notebook consists of contrived / toy examples, the data being operated on is in fact _real_! It is currently stored in the FHIR server as part of the PHDI Ingestion Pipeline and has been transformed and standardized by a number of preceding building blocks."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "dc8062d0",
+   "metadata": {},
+   "source": [
+    "To run this notebook, you'll need to configure your system to recognize the R kernel. A simple tutorial can be found here: https://irkernel.github.io/installation/. To start, make sure you've installed the latest version of R, available from the CRAN repository at https://cloud.r-project.org/index.html. Then, launch R from the terminal with the `R` command. **Important:** Do not launch R from the app--the following package installation **must** be performed from the terminal (this is because the R app does not consistently respect PATH changes set in bash profiles). Run `install.packages('IRkernel')`, then run `IRkernel::installspec()`. That's it! Your notebook software can now \"see\" the R kernel on your machine."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "eca904da",
+   "metadata": {},
+   "source": [
+    "## Packages and imports"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "93cdcd4d",
+   "metadata": {},
+   "source": [
+    "We'll begin by installing the relevant analytics packages that we'll use for our examples. We'll need:\n",
```

Comment:
You need to install the Azure related libraries as well since those are required to connect to cloud storage.

Reply:
Yeah I had that in a separate sentence but I've moved it all together to make things clearer